modules/gatsby-theme-ocular/bin/ocular.js

```diff
@@ -146,7 +146,7 @@ const commands = {
         }
 
         writeFileSync(`${DIR_PATH}/package.json`, `${JSON.stringify(UPDATED_PACKAGE_JSON, null, 2)}\n`)
-        
+
         const ocularConfig = OCULAR_CONFIG_TEMPLATE(result);
         writeFileSync(`${DIR_PATH}/ocular-config.js`, ocularConfig);
 
```

Comment:
Can all of these commented out writeFileSync calls be deleted?

Reply:
deleted